client/galaxy/scripts/mvc/tours.js

```diff
@@ -10,7 +10,7 @@
 
 define(['libs/bootstrap-tour'],function(BootstrapTour) {
 
-    var gxy_root = Galaxy === undefined ? '/' : Galaxy.root;
+    var gxy_root = typeof Galaxy === "undefined" ? '/' : Galaxy.root;
```

Comment:
is it possible to use the same check as in the other Backbone apps? 
`window.galaxy_config ? galaxy_config.root : '/'`